### ISSUE TITLE
qt5_avplay: build and packaging error

### ISSUE THREAD

**Issue by nfeske** [genodelabs/genode-world#222]
The recipes and and qt5_avplay run script need to be adjusted for the changes of genodelabs/genode#3795.

**Comment by nfeske** [comment]
Commit https://github.com/nfeske/genode-world/commit/9c5df722ea3ba765aba10a6db11502f54efcf256 is my first take on it. It does not solve it completely though:

```
../../../../repos/world/src/app/qt_avplay/gui_session_component.h:21:10: fatal error: qgenodeplatformwindow.h: No such file or directory
```

Maybe @cproc can step in?

**Comment by cproc** [comment]
Fixed in 165d65f
